edb/server/tenant.py

```diff
@@ -783,7 +783,7 @@ async def _acquire_intro_pgcon(
         try:
             conn = await self.acquire_pgcon(dbname)
         except pgcon_errors.BackendError as e:
-            if e.code_is(pgcon_errors.ERROR_INVALID_CATALOG_NAME):
+            if e.code_is(pgcon_errors.ERROR_INVALID_CATALOG_NAME) or True:
```

Comment:
Dont wanna keep this

Reply:
Oh, yeah. That was there because in the initial take, we created a totally empty database, and if the copy failed and then the remove didn't happen, we'd end up with a database that couldn't start up, so you couldn't even clean it up from edgedb. Now, though, since we still use the template, that's not a problem.